ignore.go

```diff
@@ -0,0 +1,119 @@
+package metafora
+
+import (
+	"container/heap"
+	"sync"
+	"time"
+)
+
+type ignoremgr struct {
```

Comment:
Oh, hey this code looks like the old ttl scheduler :p... sweet!  +1 on `Share Memory By Communicating`.